test/jasmine/tests/gl2d_click_test.js

```diff
@@ -180,9 +180,10 @@ describe('Test hover and click interactions', function() {
         gd = createGraphDiv();
     });
 
-    afterEach(function() {
+    afterEach(function(done) {
         Plotly.purge(gd);
         destroyGraphDiv();
+        setTimeout(done, 1000);
```

Comment:
this _appears_ to reduce the number of intermittent failures (and the number of retry attempts) _a little_.